Source/WebKit/UIProcess/DigitalCredentials/WKDigitalCredentialsPicker.h

```diff
@@ -29,9 +29,8 @@
 #if PLATFORM(IOS_FAMILY)
 #import <UIKit/UIKit.h>
 #endif
-#import <WebKit/WKWebView.h>
-#import <wtf/CompletionHandler.h>
 
+OBJC_CLASS WKWebView;
```

Comment:
```suggestion
@class WKWebView;
```

`@class` is fine here since this header already uses Obj-C.

Reply:
makes sense. Thank you!